src/server/util/transitionPage.ts

```diff
@@ -47,7 +47,7 @@ export class CookieArrayReducer implements CookieReducer {
     shortUrl: string,
   ): string[] {
     if (!cookie) return [shortUrl]
-    if (this.userHasVisitedShortlink(cookie, shortUrl)) {
+    if (cookie.includes(shortUrl)) {
```

Comment:
what's the weird inversify bug?

Reply:
even though it's declared as a member of this class, when it is `bind`ed by inversify, `this` resolves to `undefined`

Reply:
Using an arrow function avoids javascript's weird 'this' binding behavior